### ISSUE TITLE
Remembering audio and subtitles tracks for a show

### ISSUE THREAD

**Issue by CatOyona** [jarnedemeulemeester/findroid#891]
Hello!

I have multiples shows and I really like how the app looks. However I am missing a feature that could solve a problem I have: when I am watching a show, I am watching with very specifics tracks: original audio track, and subtitle track for my native language. You could tell me: set the right audio and sub favorite language in the app's settings. But it doesn't solve my problem because the track I use is not necesarry the first one coming up for the selected language. Let me illustrate: I watch my shows with french subs, so my preferred sub language in the settings is French. However, many shows use forced and full subtitles, and the app selects the forced by default so I have to manually choose the full subs track for every episode, which is kind of annoying.

What I'd like is the feature that already exists for the normal Jellyfin app, which is the 'set audio/subtitle track based on previous item' setting. It basically remember what audio and subs track number is used for every show, separately.

That's about it, I really hope that I'll see this feature in the app because it really looks so much better than the normal Jellyfin app, and i also hope it isn't too much complicated to implement.

Have a nice day!

**Comment by foux** [comment]
I agree with the issue, but I think it's more a server issue than an issue of the player

**Comment by foux** [comment]
Well, let me correct that, it's definitely a player issue, as it doesn't respect settings on the server
